src/components/usa-date-picker/usa-date-picker.stories.js

```diff
@@ -0,0 +1,29 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-date-picker.twig";
+import Data from "./usa-date-picker.json";
+
+export default {
+  title: "Components/Date Picker",
+  argTypes: {
+    default_date: {
+      control: "text",
```

Comment:
Do these controls do anything?

Reply:
These would modify the calendar pop-up but now the pop up doesn't seem to be loading at all on storybook? It was working on refresh before.

Here's a screenshot from the latest federalist build
![image](https://user-images.githubusercontent.com/25211650/151826643-e90d063b-8708-4530-af95-a4f101806b21.png)